packages/react-native-worklets/src/shareables.ts

```diff
@@ -165,6 +165,13 @@ function makeShareableCloneRecursiveNative<T>(
   if (isHostObject(value)) {
     return cloneHostObject(value, shouldPersistRemote);
   }
+  if (isPlainJSObject(value) && value.__init) {
```

Comment:
Why don't we use `'__init' in value` instead? Which one is better?

Reply:
imo  it doesn't matter,  `'__init' in value` should also be ok, although it doesn't check whether there is any value under the `__init` key